sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/InsertIntoHiveTable.scala

```diff
@@ -50,7 +50,7 @@ case class InsertIntoHiveTable(
   @transient val sc: HiveContext = sqlContext.asInstanceOf[HiveContext]
```

Comment:
Existing:  Calling a HiveContext `sc` is pretty confusing.  Also since this is a convenience cast it should be a `def` so as to not take up extra space in the object.